BrightCoders-Institute/s10a1-proyecto-final-team-01-s10a1-proyecto-final
# Property Management For Hosts

# Description

- Generated completely functional Accommodations controller and views.  
- Accommodations management with Google Maps API, Active Storage and Action Text features.
- Users creations and additions bugs fixes.
- Tests model, controller and views deletion.

## Testing

- Run the corresponding Rails migrations in order to properly update the local database.
- Ask the team for the application's API credentials and master key files.
- Put the credentials files into the config folder.
- Run the application with either rails server or ./bin/dev commands.
- Navigate to /accommodations route in order to check the corresponding CRUD.
